.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -1347,6 +1348,7 @@ tt
 ttf
 ttype
 tw
+Twr
```

Comment:
bunch of crappy words are leaking to our whitelist, lets exclude .ci/openjdk14-excluded.files completely from spell-checker

Reply:
Done.